README.rst

```diff
@@ -88,17 +88,29 @@ Installation
 
 #. Include these context processors::
 
+    Django 1.7::
     TEMPLATE_CONTEXT_PROCESSORS = (
         "django.core.context_processors.request",
         "treenav.context_processors.treenav_active",
     )
 
+    Django >= 1.8::
```

Comment:
In the rendered version of the README, the 'Django 1.7::` looks like it's part of the code that should be entered in the settings file. Not a big deal, but if you can figure out how to make that look cleaner (or prepend those with comments, so if people do enter them, they don't cause errors)